### PR TITLE
less informal writing around composition / defaults

### PR DESCRIPTION
**Pull request by mshvartsman**
## Motivation

Rewriting some text that can perhaps read as more adversarial than intended. 

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebookresearch/hydra/blob/master/CONTRIBUTING.md)?

Yes.

## Test Plan

Built the website and viewed locally, looks fine.

### CODE REVIEW

File: website/docs/tutorials/basic/your_first_app/6_composition.md [hunk 1]
```diff
@@ -3,15 +3,13 @@ id: composition
 title: Putting it all together
 ---
 
-The product manager had an idea:
-She wants my_app to support 3 schemas on 2 different databases and 2 kinds of UI for start. More will come later.
+As software gets more complex, we resort to modularity and composition to keep it manageable. 
+We can do the same with configs: suppose we want our working example to support multiple databases, with
+multiple schemas per database, and different UIs. We wouldn't write a separate class
+for any permutation of db, schema and UI, so we shouldn't write separate configs either. We use 
+the same solution in configuration as in writing the underlying software: composition.
```
Comment (omry) [inline website/docs/tutorials/basic/your_first_app/6_composition.md:10]:
```suggestion
As software gets more complex, we resort to modularity and composition to keep it manageable. 
We can do the same with configs: suppose we want our working example to support multiple databases, with
multiple schemas per database, and different UIs. We wouldn't write a separate class
for each permutation of db, schema and UI, so we shouldn't write separate configs either. We use 
the same solution in configuration as in writing the underlying software: composition. 
```

File: website/docs/tutorials/basic/your_first_app/5_defaults.md [hunk 2]
```diff
@@ -3,8 +3,7 @@ id: defaults
 title: Selecting defaults for config groups
 ---
 
-After office politics, you decide that you want to use MySQL by default.
-You no longer want to type `+db=mysql` every time you run your application.
+Suppose that you want to use MySQL by default -- you no longer want to type `+db=mysql` every time you run your application.
```
Comment (omry) [inline website/docs/tutorials/basic/your_first_app/5_defaults.md:6]: I actually like the office politics bit.

Comment (mshvartsman) [reply website/docs/tutorials/basic/your_first_app/5_defaults.md]: fine, reverted

**Comment by omry** [comment]
awesome.

**Pull request merged**